pages/docs/community/styleguide/links.md

```diff
@@ -0,0 +1,35 @@
+# Link Style Guide 📃
```

Comment:
I don't think we should be using emojis in style guides also please add
```suggestion
---
title:
description:
weight:
---
# Link Style Guide 📃
```
just like all other docs have. You can check out other docs in docs folder

Reply:
agreed, we don't use emojis in docs so please remove them :)

Reply:
I thought emojis will make it more user friendly :)